src/inmanta/parser/plyInmantaLex.py

```diff
@@ -132,46 +130,22 @@ def t_JCOMMENT(t: lex.LexToken) -> None:  # noqa: N802
     pass
 
 
-def t_begin_mls(t: lex.LexToken) -> lex.LexToken:
-    r'["]{3}'
-    t.lexer.begin("mls")
-    t.type = "MLS"
-
-    lexer = t.lexer
-
-    end = lexer.lexpos - lexer.linestart + 1
-    (s, e) = lexer.lexmatch.span()
-    start = end - (e - s)
-
-    t.value = LocatableString("", Range(lexer.inmfile, lexer.lineno, start, lexer.lineno, end), lexer.lexpos, lexer.namespace)
-
-    return t
-
-
-def t_mls_end(t: lex.LexToken) -> lex.LexToken:
-    r'.*["]{3}'
-    t.lexer.begin("INITIAL")
-    t.type = "MLS_END"
-    value = t.value[:-3]
-
+def t_MLS(t: lex.LexToken) -> lex.LexToken:
+    r'"{3}([\s\S]*?)"{3}'
+    t.value = bytes(t.value[3:-3], "utf-8").decode("unicode_escape")
```

Comment:
Why add unicode escape decoding? This actually changes behavior.

Reply:
I did this  for consistency as it was also done like this for strings. I will remove it.